### PR TITLE
[MAINTENANCE] Cut over analytics to prod

### PR DESCRIPTION
**Pull request by tyler-hoffman**
I think this is all that's needed for V1-333, but maybe there's more.

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: great_expectations/analytics/config.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class _EnvConfig(BaseSettings):
 
     gx_posthog_debug: bool = False
     gx_posthog_host: HttpUrl = "https://posthog.greatexpectations.io"  # type: ignore[assignment] # default will be coerced
-    gx_posthog_project_api_key: str = ""
+    gx_posthog_project_api_key: str = "phc_ph6ugZ1zq94dli0r1xgFg19fk2bb1EdDoLn9NZnCvRs"
```
Comment (billdirks) [inline great_expectations/analytics/config.py:18]: I think we can get rid of `_gx_cloud_dev_posthog_project_api_key` altogether with this change.

File: great_expectations/analytics/config.py [hunk 2]
```diff
@@ -45,6 +45,7 @@ class Config(GenericModel):
     data_context_id: Optional[UUID] = None
     oss_id: Optional[UUID] = None
     cloud_mode: bool = False
+    service: str = "gx-core"
```
Comment (billdirks) [inline great_expectations/analytics/config.py]: I'm not sure we need this; is it consumed somewhere? We do send this service value with every event we send to posthog. That is configured in [base event class here.](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/analytics/base_event.py#L75)

**Review by billdirks — COMMENTED** [review]
Looks good. Thanks for tracking this down. I have a few cleanup comments.

File: great_expectations/analytics/config.py (hunk 2 above)
Comment (tyler-hoffman) [reply great_expectations/analytics/config.py]: Thanks, I missed that!

**Review by billdirks — APPROVED** [review]
LGTM, thanks!

**Pull request merged**